tools/mo/openvino/tools/mo/front/tf/prelu.py

```diff
@@ -9,46 +9,6 @@
 from openvino.tools.mo.graph.graph import Graph
 from openvino.tools.mo.middle.pattern_match import check_node_usages_out_of_match
 
-
-class PReLUPatternFuse(FrontReplacementSubgraph):
```

Comment:
Changes were rollbacked. Please note, that in case when old MO transformations working we can face with problem with operation names. For my particular case there is no problem, since mo didn't catch pattern, but for other cases it can be applicable.